dev/TeachingTip/InteractionTests/TeachingTipTests.cs

```diff
@@ -178,6 +178,37 @@ public void TipCanFollowTarget()
             }
         }
 
+        [TestMethod]
+        public void TipFollowsTargetOnWindowResize()
+        {
+            using (var setup = new TestSetupHelper("TeachingTip Tests"))
+            {
+                elements = new TeachingTipTestPageElements();
+                foreach (TipLocationOptions location in Enum.GetValues(typeof(TipLocationOptions)))
+                {
+                    SetTeachingTipLocation(location);
+
+                    ScrollTargetIntoView();
+                    Wait.ForIdle();
+                    OpenTeachingTip();
+                    double initialTipVerticalOffset = GetTipVerticalOffset();
+                    double initialScrollViewerVerticalOffset = GetScrollViewerVerticalOffset();
+
+                    ScrollBy(10);
+                    WaitForOffsetUpdated(initialScrollViewerVerticalOffset + 10);
+                    Equals(GetTipVerticalOffset(), initialTipVerticalOffset);
+
+                    //Unmaximize then maximize the window, to force a window size changed event.
+                    KeyboardHelper.PressKey(Key.Down, ModifierKey.Windows);
+                    Wait.ForIdle();
+                    KeyboardHelper.PressKey(Key.Up, ModifierKey.Windows);
```

Comment:
>KeyboardHelper [](start = 20, length = 14)

If this test were to throw an exception for some reason after unmaximizing but before re-maximizing, could it leave the test app in a bad state for the following tests?

Reply:
If the test fails the test harness kills the test app and restarts it, so I think we are protected from that.

Reply:
Cool.